src/style.css

```
@@ -14,28 +14,10 @@ body {
   background: url(images/star.svg) center no-repeat;
 }
 
-.stars--1 .stars__star:nth-child(1) {
-  background-image: url(images/star-active.svg);
-}
-
-.stars--2 .stars__star:nth-child(1),
-.stars--2 .stars__star:nth-child(2) {
-  background-image: url(images/star-active.svg);
-}
-
-.stars--3 .stars__star:nth-child(1),
-.stars--3 .stars__star:nth-child(2),
-.stars--3 .stars__star:nth-child(3) {
-  background-image: url(images/star-active.svg);
-}
-
-.stars--4 .stars__star:nth-child(1),
-.stars--4 .stars__star:nth-child(2),
-.stars--4 .stars__star:nth-child(3),
-.stars--4 .stars__star:nth-child(4) {
-  background-image: url(images/star-active.svg);
-}
-
-.stars--5 .stars__star:nth-child(1n) {
+.stars--1 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:nth-child(-n+4),
+.stars--5 .stars__star:nth-child(-n+5) {
```

```suggestion
.stars--5 .stars__star {
```
since there only five stars - no need to provide `nth-child` in this case